handbook/company.md

```diff
@@ -32,14 +32,14 @@ Fleet's values are a set of five ideals adopted by everyone on the team.  They d
 4. 🔵 Objectivity
 5. 🟣 Openness
 
-When a new team member joins Fleet, they adopt the values, from day 1.  This way, even as the company grows, everybody knows what to expect from the people they work with. Having a shared mindset keeps us quick and determined.
+When a new team member joins Fleet, they adopt the values, from day one.  This way, even as the company grows, everybody knows what to expect from the people with whome they work. Having a shared mindset keeps us quick and determined.
```

Comment:
I think this is more readable before the edit.

Reply:
We have switched to writing out numbers under 10. The second edit was because the sentence ended with a preposition.

Reply:
@Desmi-Dizney :+1: I understand about numbers.  I'm referring to https://www.google.com/search?q=whome  😄

Reply:
I could have been more explicit and saved us some time.

I think it's probably because I'm not convinced we should always be eliminating dangling prepositions, but I don't have enough conviction about that to even mention it.  I think you should stick with that rule.  So I think that's why I ended up communicating vaguely.  Sorry about that.

Reply:
Thanks for the clarification. I'm always up for feedback. I'll drop the "e" from "whome" to be more modern.